manual_user_install.sh

```diff
@@ -0,0 +1,235 @@
+    pip3 install PyYAML
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install PyAudio
+
+    # install speech to text (stt)
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-microphone-plugin-alsa
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-stt-plugin-server
+
+    # install text to speech (tts)
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-mimic3-server
+    sudo apt install -y espeak-ng
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-plugin-piper
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-tts-server-plugin
+
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-config
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-utils
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-bus-client
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-plugin-manager
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-workshop
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-lingua-franca
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-cli-client
+
+    # install phal components
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-connectivity-events
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-system
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-ipgeo
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-PHAL-plugin-oauth
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-dashboard
+    pip3 install git+https://github.com/OpenVoiceOS/ovos-phal-plugin-alsa
+
+    # install required skills
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-volume
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-fallback-unknown
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-stop
+    pip3 install git+https://github.com/OpenVoiceOS/skill-alerts
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-personal
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-naptime
+    pip3 install git+https://github.com/OpenVoiceOS/skill-ovos-date-time
+
+    # You can uncomment these lines if the deprecation notices are flooding your logs or slowing the system
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/fingerprinting.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/configuration.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/ovos_service_api.py
+    #sed -i '/\@deprecated/d' $HOME/.local/lib/python3.9/site-packages/ovos_utils/signal.py
+
+    if [[ ! -d $HOME/.config/mycroft ]]; then
+        mkdir -p $HOME/.config/mycroft
+    fi
+    cp $HOME/raspbian-ovos/stage-ovos/02-ovos/files/mycroft.conf $HOME/.config/mycroft/
```

Comment:
Suggestion: don't assume this is checked out into the home directory. Set this at the top of the file

```
SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

```suggestion
    cp $SCRIPT_DIR/raspbian-ovos/stage-ovos/02-ovos/files/mycroft.conf $HOME/.config/mycroft/
```

Reply:
I am drinking from a fire hose at the moment. :) Your suggestions are welcome and they are right on. I have stopped working on the script (after adding automatic installation of the pre-requisite software) while I sort out exactly how the mycrof.conf interacts with the plugins. Then that will be built into the script. It will take me a few days to get that right. In the meantime, the script is sufficient to get ovos to tell you the time and temperature, but won't help you get to the system improvements that we're trying to enable unless you already understand the system.  If you look in /var/log/syslog, you'll see that the interaction between the plugins and the mycroft.conf files (~/.local/config/mycroft/mycroft.conf and ~/.local/lib/python***/site_packages/ovos_config/mycroft.conf) isn't smooth. builderjer and JarbasAI are drinking from fire hoses too, just on more important issues. Please stay involved. There's plenty of water to go around! builderjer's original idea was that this script could help him quickly and easily try out different configurations.  I see it as an ideal training vehicle for new users like me.